### DISCUSSION TITLE
Runtime error

### DISCUSSION THREAD

**Discussion by seng1218** [vladmandic/sdnext#3377] [General]
Hi vladmandic, 
it's me again, I've encounter 1 specific problem. It showed up randomly this error message. 

:\Stable diffusion\automatic\venv\lib\site-packages\torchvision\transforms\functional.py:282: RuntimeWarning: invalid value encountered in cast
  npimg = (npimg * 255).astype(np.uint8)

After this message the image generate will be pitch black. 

what should i do about it? Is there anything I need to fix?

Thanks in advance.

**Comment by vladmandic** [comment]
this is one of the most discussed errors ever - please search.
in short, its about model precision vs your configured gpu precision. and since i don't know either of those, not much i can can say.
